packages/pcaudiolib.rb

```diff
@@ -0,0 +1,30 @@
+require 'package'
+
+class Pcaudiolib < Package
+  description 'PCaudiolib is a portable C audio library.'
+  homepage 'https://github.com/espeak-ng/pcaudiolib'
+  version '1.1'
+  license 'GPL-3'
+  compatibility 'all'
+  source_url 'https://github.com/espeak-ng/pcaudiolib/archive/refs/tags/1.1.tar.gz'
+  source_sha256 '699a5a347b1e12dc5b122e192e19f4db01621826bf41b9ebefb1cbc63ae2180b'
+
+  depends_on 'alsa_lib'
+
+  def self.build
+    system './autogen.sh'
+    system "env CFLAGS='-O2 -pipe -flto=auto -fuse-ld=gold' \
+                LDFLAGS='-flto=auto' \
+            ./configure #{CREW_OPTIONS} \
+              --without-pulseaudio \
+              --with-alsa \
+              --without-qsa \
+              --without-coreaudio \
+              --without-oss"
```

Comment:
Why so many withouts?

Reply:
Coreaudio is a macos thing, I don't know what qsa and oss are (perhaps windows, i'm not sure), alsa worked and pulseaudio had too many depenedencies for the function of the package it seemed. You could add pulseaudio as a dependency and do `--with-pulseaudio` but I chose not to.